From ee5947ca010a7d4e1206579b5450566e2601f84f Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Sun, 16 Jun 2019 13:50:06 -0700
Subject: [PATCH] Prevent overlapping storage of global variables

This variable has the same name as `struct xfrm_filter filter` in
ip/ipxfrm.c
---
 ip/ipmroute.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ip/ipmroute.c b/ip/ipmroute.c
index 6239e4c3..65b878bb 100644
--- a/ip/ipmroute.c
+++ b/ip/ipmroute.c
@@ -44,7 +44,7 @@ static void usage(void)
 	exit(-1);
 }
 
-struct rtfilter {
+static struct rtfilter {
 	int tb;
 	int af;
 	int iif;
-- 
2.20.1

